plugins/http_plugin/http_plugin.cpp

```diff
@@ -541,6 +541,22 @@ namespace eosio {
             throw;
          }
       }
+
+#define CALL(api_name, call_name, http_response_code) \
```

Comment:
As we only have 1 endpoint to add, and a `#define` in a poor place that would imply some amount of scoping to the method body (which doesn't exist), can we just eliminate the `#define` by interpolating it out ?

Reply:
I added this macro in preparation for `/v1/node/get_activated_plugins` in following patch, but it would be all right to add it in that.